src/webview/js/js.js

```diff
@@ -81,6 +83,31 @@ if (!Element.prototype.closest) {
   };
 }
 
+/* Polyfill String#startsWith. Native in Mobile Safari 9, Chrome 49. */
+if (!String.prototype.startsWith) {
+  // $FlowFixMe (polyfill)
+  Object.defineProperty(String.prototype, 'startsWith', {
```

Comment:
Huh interesting. What's the reasoning for choosing between this and `String.prototype.startsWith = ...`, like you do with `includes` below?